assets/app/view/game/part/track.rb

```diff
@@ -50,21 +88,67 @@ def render
             .flat_map { |path| path_indexes[path].map { |i| [path, i] } }
             .sort_by { |_, index| index || -1 }
 
-          sorted.map do |path, index|
+          # track outline
+          # this has to be done in a separate pass to render connections ("switches")
+          # between tracks correctly
+          passes = sorted.map do |path, index|
             props = {
-              color: value_for_index(index, :color, path.track),
+              color: value_for_index(index, :color, path.gauge),
               width: width_for_index(path, index, path_indexes),
-              dash: value_for_index(index, :dash, path.track),
+              dash: value_for_index(index, :dash, path.gauge),
             }
 
+            border_props = BORDER_PROPS[path.gauge]
+            inner_props = INNER_PROPS[path.gauge]
+
+            if !path.stub? && !path.offboard
+              h(TrackNodePath, tile: @tile, path: path, region_use: @region_use,
+                               pass: 0, border_props: border_props, inner_props: inner_props, **props)
+            end
+          end
```

Comment:
missing compact?

Reply:
Yep, no idea why it didn't cause a problem.